Gruntfile.js

```diff
@@ -143,15 +147,113 @@ module.exports = function (grunt) {
+                            platform: 'Windows XP'
+                        }, {
+                            browserName: 'opera',
+                            version: '12.12',
+                            platform: 'Windows XP'
+                        }, {
+                            browserName: 'safari',
+                            version: '5.1',
+                            platform: 'Windows 7'
+                        },
+
+                        // MAC
+                        {
+                            browserName: 'firefox',
+                            version: '4.0',
+                            platform: 'OS X 10.6'
+                        }, {
+                            browserName: 'chrome',
+                            version: '27.0',
+                            platform: 'OS X 10.6'
+                        }, {
+                            browserName: 'safari',
+                            version: '5.1',
+                            platform: 'OS X 10.6'
+                        },
+
+                        // LINUX
+                        {
+                            browserName: 'firefox',
+                            version: '4.0',
+                            platform: 'Linux'
+                        }, {
+                            browserName: 'chrome',
+                            version: '26.0',
+                            platform: 'Linux'
+                        }, {
+                            browserName: 'opera',
+                            version: '12.15',
+                            platform: 'Linux'
+                        }
+
+                        /*
+                        // Android
+                        ,{
+                            browserName: 'android',
+                            version: '4.1',
+                            deviceName: 'Android Emulator',
+                            platform: 'Linux'
+                        },
+                        // PocketMAC
+                        {
+                            browserName: 'iphone',
+                            version: '7.0',
+                            deviceName: 'iPhone Simulator',
+                            platform: 'OS X 10.10'
+                        }, {
+                            browserName: 'iphone',
+                            version: '7.0',
+                            deviceName: 'iPad Simulator',
+                            platform: 'OS X 10.10'
+                        }
+                        */
+                    ],
+                    testname: "Sauce Labs Test for CraftyJS",
+                    build: process.env.TRAVIS_BUILD_NUMBER,
+                    tags: [ process.env.TRAVIS_BRANCH ],
+                    sauceConfig: {
+                        "recordVideo": false,
+                        "recordScreenshots": false,
+                        "disablePopupHandler": true
+                    },
+                    throttled: 2,
```

Comment:
After you create your open sauce account, check on the main page on the left how many parallel tests you can have running (I can only have 2 because of my limited account) -> change this number to reflect the max amount of parallel tests